compiler/rustc_passes/src/reachable.rs

```diff
@@ -1,9 +1,15 @@
-// Finds items that are externally reachable, to determine which items
-// need to have their metadata (and possibly their AST) serialized.
-// All items that can be referred to through an exported name are
-// reachable, and when a reachable thing is inline or generic, it
-// makes all other generics or inline functions that it references
-// reachable as well.
+//! Finds local items that are externally reachable, to determine which items
+//! need to have their metadata (and possibly their AST) serialized.
```

Comment:
Why does it talk about "metadata and AST"? The main concern for this computation seems to be codegen -- otherwise, inlining wouldn't matter, right?

Reply:
Okay, will update this to
```
//! Finds local items that are externally reachable, which means that other crates need access to
//! their MIR.
```

Reply:
Hm, this does not sound entirely right though... it may be either the MIR or the generated IR, right?

Reply:
I have now made it
```
//! Finds local items that are externally reachable, which means that other crates need access to
//! their compiled machine code or their MIR.
```